redisjson/README.md

```diff
@@ -0,0 +1,70 @@
+<!--
+Copyright (c) 2014 - 2020 YCSB contributors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you
+may not use this file except in compliance with the License. You
+may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+implied. See the License for the specific language governing
+permissions and limitations under the License. See accompanying
+LICENSE file.
+-->
+
+## Quick Start
+
+This section describes how to run YCSB on RedisJson. 
+
+### 1. Start Redis with the RedisJSON module loaded (cf. https://oss.redislabs.com/redisjson/#building-and-loading-the-module)
+
+### 2. Install Java and Maven
+
+### 3. Set Up YCSB
+
+Git clone YCSB and compile:
+
+    git clone http://github.com/brianfrankcooper/YCSB.git
+    cd YCSB
+    mvn -pl site.ycsb:redisjson-binding -am clean package
+	
+### 4. Set up JRedisJSON
+	In a separate directory ( the JAR will be installed in your local
+	maven repo ):
+	git clone -b reflection https://github.com/RedisJSON/JRedisJSON.git 
```

Comment:
please rely on a released artifact and not build-from-source instructions.

Reply:
Unfortunately not possible in this case.

Reply:
Why not? Can'd the RedisJSON project make a release?

Reply:
Cf. above comment.

Reply:
Fixed the README as JRedisJSON now includes the reflective patch as part of its 2.0 master branch.